### PR TITLE
Add C language code sample for tests

### LINKED ISSUE #25: Add C language code sample for tests
Add C language code samples for tests on analyzer and scanner

### PR DESCRIPTION
**Pull request by miles170**
Closes #25.

### CODE REVIEW

File: test/fixtures/code_samples/go.mod [hunk 1]
```diff
@@ -0,0 +1 @@
+// Exclude this directory from go
```
Comment (JoaoDanielRufino) [inline test/fixtures/code_samples/go.mod:1]: Could you delete this file ?

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]
Good work, thanks for creating this PR.

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]
The file `test/fixtures/code_samples/go.mod` is still there, could you **DELETE** the file ?

**Comment by miles170** [comment]
If I delete `test/fixtures/code_samples/go.mod`, `go list ./...` will throw `package github.com/JoaoDanielRufino/gcloc/test/fixtures/code_samples: C source files not allowed when not using cgo or SWIG: main.c`

**Review by JoaoDanielRufino — APPROVED** [review]
Good work!

**Pull request merged**